test/JDBC/input/BABEL-3952-vu-prepare.sql

```diff
@@ -0,0 +1,374 @@
+-- Test with null datepart
+-- Should Throw Error - 'syntax error at or near "null"'
+-- VIEW DATE_BUCKET_vu_prepare_v1 DATE_BUCKET_vu_prepare_v2 should not get created.
+CREATE VIEW DATE_BUCKET_vu_prepare_v1 AS (
```

Comment:
Added testcases where each parameter is NULL , text , decimal values  , expressions if not already added. Add testcase where it contains the upper limit and lower limit for each date/time datatype.

Reply:
Added test case. however expressions will be considered as string in date_bucket() function unless explicitly casted to a date value datatype.